tools/appimage.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+# The MIT License (MIT)
+# Copyright (c) 2016 Simon Peter
+
```

Comment:
If this was based on existing script covered by an existing MIT license, Please attribute here as well, give the location of the license (and mention the type).

---
In reply to: [75977883](https://github.com/PowerShell/PowerShell/pull/2027#discussion_r75977883) [](ancestors = 75977883)